bench/src/bench.rs

```diff
@@ -115,15 +119,31 @@ fn civil_datetime_to_instant_static(c: &mut Criterion) {
 /// human readable representation (i.e., what you might see on a clock) in a
 /// specific time zone.
 fn instant_to_civil_datetime_static(c: &mut Criterion) {
+    instant_to_civil_datetime_static_with_specific_tz(
+        c,
+        TZ_NEW_YORK,
+        1719755160,
+    );
+    instant_to_civil_datetime_static_with_specific_tz(
+        c,
+        TZ_SHANGHAI,
+        1719711960,
+    );
+}
+
+fn instant_to_civil_datetime_static_with_specific_tz(
```

Comment:
This function should probably be defined within `instant_to_civil_datetime_static`. And since the scope is narrower there, you can just call it `define`.

Reply:
> you can just call it define.

Sorry, I don't understand it. Do you mean rename the funcation name to `instant_to_civil_datetime_static_define`?

Reply:
No. `define`. But you need to make it an inner function first:

```rust
fn instant_to_civil_datetime_static(...) {
    fn define(...) {
        ...
    }
    define("America/New_York", ...);
    define("Asia/Shanghai", ...);
}
```